contracts/utils/Checkpoints.sol

```diff
@@ -15,21 +16,233 @@ import "./math/SafeCast.sol";
  * _Available since v4.5._
  */
 library Checkpoints {
-    struct Checkpoint {
```

Comment:
Are we removing this one? Technically it should be kept for backwards compatibility...

Reply:
This one is replaced by `Checkpoint224`. 

I asked that question a while back, and we said that since this is an internal structure, that devs are not expected to interact with directly, it was ok to rename it.

Ideally I would to something like `type Checkpoint is Checkpoint224;` ... but that is only authorized for value-types.